.github/workflows/coffee.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container: 
-      image: golang:1.20
+      image: golang:1.19
```

Comment:
downgraded to 1.19 due to https://issues.redhat.com/browse/RHELPLAN-167618
so keeping the go version compatible with other components (including e2e-tests)